agimus_controller/agimus_controller/mpc_data.py

```diff
@@ -19,7 +17,7 @@ class OCPResults:
 class OCPDebugData:
     # Debug data
     result: OCPResults = OCPResults()
-    references: list[TrajectoryPoint] = field(default_factory=list)
+    references: T.Dict[str, npt.NDArray[np.float64]] = field(default_factory=dict)
```

Comment:
For the sake of performance, a list of tuple (name, value) would be more efficient.